x-pack/test/cloud_security_posture_functional/pages/findings.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import type { FtrProviderContext } from '../ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const pageObjects = getPageObjects(['common', 'findings']);
+  const retry = getService('retry');
+  const supertest = getService('supertest');
+  const log = getService('log');
+
+  /**
+   * This is required prior to indexing findings
+   */
+  const waitForPluginInitialized = (): Promise<void> =>
+    retry.try(async () => {
+      log.debug('Check CSP plugin is initialized');
+
+      const response = await supertest
+        .get('/internal/cloud_security_posture/status?check_initialized=true')
+        .expect(200);
+
+      expect(response.body).to.eql({ initialized: true });
+
+      log.debug('CSP plugin is initialized');
+    });
+
+  describe('Findings Page', () => {
+    before(async () => {
+      await waitForPluginInitialized();
+      await esArchiver.load('x-pack/test/functional/es_archives/findings');
```

Comment:
`esArchiver` loads here not just data.json, but also mapping.json. 
The integration can be installed using the API, which will add the mappings and trigger whatever is necessary for our solution to work.
There is no doubt this API takes a long time, but is it the only reason we chose the current approach?
Also, I wonder how the mapping.json file will be maintained in case something will be changed in the original mapping.

Reply:
isn't indexing via API / archive file really the same in terms of compatibility with cloudbeat? 
because any change to [`Findings`](https://github.com/elastic/cloudbeat/blob/80497b54899f1bf46646813f069f5a4bcff04566/evaluator/evaluator.go#L44-L47) will require changes to the data we want to index.

so eventually you need that "demo data" to match the latest integration index template, and doing it with an archive file just seems faster? 

if i'm misisng smth about how the compatibility issue is different when indexing via API, please LMK.

Reply:
changed to indexing via API